internal/handler/userHandler.go

```diff
@@ -82,24 +93,36 @@ func (h *Handler) Save(context *gin.Context) {
 // @Description Find user by provided id
 // @Tags Users
 // @Produce json
-// @Param id path int true "User id"
-// @Success 201 {object} response.UserResponseDto "User Found successfully"
-// @Failure 422 {object} response.ErrorDto "Id in the wrong format"
+// @Param id path string true "User id"
+// @Failure 422 {object} response.ErrorDto "ID in the wrong format"
 // @Failure 404 {object} response.ErrorDto "User not found"
+// @Failure 500 {object} response.ErrorDto "Failed to connect to database"
 // @Router /users/{id} [get]
 func (h *Handler) FindById(context *gin.Context) {
 	id := context.Param("id")
 	logger.Infof("Finding user by id %s", id)
 	user, err := h.service.GetUserById(context, id)
+
 	if err != nil {
-		logger.Errorf("Error while finding user by id %s. %v", id, err)
+		logger.Errorf("Error while connecting to database. %v", err)
```

Comment:
Same as other comment, this log message should not be database specific.

Reply:
fixed